arch/arm64/include/asm/page_utils.h

```diff
@@ -0,0 +1,78 @@
+#ifndef __PAGE_UTILS_ARM64_H_INCLUDE__
+#define __PAGE_UTILS_ARM64_H_INCLUDE__
+
```

Comment:
This can be merged to arch/arm64/include/asm/page.h.
With:
#ifdef __ASSEMBLY__
#endif

Reply:
Not yet. Because of compiling errors.